app/models/businessTaxReliefs/OtherReliefs.scala

```diff
@@ -16,9 +16,22 @@
 
 package models.businessTaxReliefs
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{JsObject, Json, OWrites, Writes}
+
+case class OtherReliefs(qualifyingLoanInterestPayments: Option[BigDecimal],:
+                        postCessationTradeReliefAndCertainOtherLosses: Option[BigDecimal],
+                        nonDeductableLoanInterest: Option[BigDecimal])
 
-case class OtherReliefs(qualifyingLoanInterestPayments: Seq[QualifyingLoanInterestPayments])
 object OtherReliefs {
-  implicit val formats: OFormat[OtherReliefs] = Json.format[OtherReliefs]
+
+  private def nestedReliefClaimedObj[A: Writes](value: Option[A])(key: String): JsObject =
+    value.map(v => Json.obj(key -> Json.arr(Json.obj("reliefClaimed" -> v)))).getOrElse(Json.obj())
+
+  implicit val writes: OWrites[OtherReliefs] = {
+    case OtherReliefs(q, p, n) =>
+      nestedReliefClaimedObj(q)("qualifyingLoanInterestPayments") ++
+      nestedReliefClaimedObj(p)("postCessationTradeReliefAndCertainOtherLosses") ++
+      nestedReliefClaimedObj(n)("nonDeductableLoanInterest")
```

Comment:
This one isn't within an array; so this is the odd one out. This should just be sent as a JSON Object (if a value exists).